src/wiki/ventoy.md

```diff
@@ -0,0 +1,42 @@
+# Creating a Multi-OS Bootable Drive (with Ventoy)
+In this article, you will learn how to create a multi-OS bootable USB drive. using Ventoy 
+
+Ventoy is a tool that allows you to create a bootable USB drive, with multiple operating systems or recovery tools. Unlike other software, Ventoy allows you to simply store the ISO files **directly** on the USB. This can be really helpful if you have a suite of tools that you often use and you don't want to use a different USB drive for each one of them.
```

Comment:
You don't need a comma here
```suggestion
Ventoy is a tool that allows you to create a bootable USB drive with multiple operating systems or recovery tools. Unlike other software, Ventoy allows you to simply store the ISO files **directly** on the USB. This can be really helpful if you have a suite of tools that you often use and you don't want to use a different USB drive for each one of them.
```